modules/core/src/index.js

```diff
@@ -1,16 +1,7 @@
-import * as path from './lib/path/path';
-
-// PATH
-export {path};
-
 // FILE READING AND WRITING
 export {fetchFile} from './lib/fetch/fetch-file';
 export {readFileSync} from './lib/fetch/read-file';
 export {writeFile, writeFileSync} from './lib/fetch/write-file';
-export {
```

Comment:
While these were marked as experimental is it worth an upgrade guide mention?

Reply:
These were never published to prod.